lib/baldr/version.rb

```diff
@@ -1,3 +1,3 @@
 module Baldr
-  VERSION = '0.10.0'
-end
\ No newline at end of file
+  VERSION = '0.10.1'
```

Comment:
@prajakta-tambe I usually increment the middle digit. I'm probably doing it wrong. @scottkf ?